conf/pipeline/sarek/uppmax.config

```
@@ -0,0 +1,17 @@
+// sarek/uppmax specific profile config
+
+params {
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
+}
+
+if (hostname ==~ "i.*") {
+  singularity {
+      cacheDir = "${params.containers_path}"
```

```suggestion
      cacheDir = params.containers_path
```
I think you maybe need to set a default value for this as well, to avoid warnings on other systems? Not sure, maybe not necessary.

---

That should only happen on irma, so we should be fine with that.